modules/preview-react/table/lib/TableCell.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {Grid, GridProps} from '@workday/canvas-kit-react/layout';
+import {createComponent} from '@workday/canvas-kit-react/common';
+
+export interface TableCellProps extends GridProps {
+  headers?: string;
```

Comment:
does this need `colspan` and `rowspan` ? Referencing: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td

Reply:
So colspan and rowspan are attributes on the `td` and `th` however, since `Table.Row` is built on `Grid`, that is how those will be made to span rows or columns since colspan and rowspan are just visual attributes.